install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1055,19 +1055,41 @@ hubble:
   #     enabled: true
   #     http:
   #       urlQuery: true
+  #       headers:
+  #         allow:
+  #           - traceparent
+  #           - tracestate
+  #           - Cache-Control
   #     kafka:
   #       apiKey: false
   #
+  # or with defining a deny list for the http headers:
+  #
+  #
+  #   redact:
+  #     enabled: true
+  #     http:
+  #       headers:
+  #         deny:
+  #           - Authorization
+  #           - Proxy-Authorization
+  #
   # You can specify the options from the helm CLI:
   #
   #   --set hubble.redact.enabled="true"
   #   --set hubble.redact.http.urlQuery="true"
+  #   --set hubble.redact.http.headers.allow="traceparent,tracestate,Cache-Control"
   #   --set hubble.redact.kafka.apiKey="false"
   #
+  #   A deny list for the https headers is also supported (cannot be used along with the allow list):
+  #   --set hubble.redact.http.headers.deny="Authorization,Proxy-Authorization"
   redact:
     enabled: false
     http:
         urlQuery: false
+        headers:
+          allow: []
+          deny: []
```

Comment:
I think the `redact` docstring here is getting too beefy and confusing here, what with the caveats and all. At this point I would suggest we instead move the appropriate bits to the individual options (e.g. document `hubble.redact.http.headers.allow` with its own docstring), and keep the main `redact` short and descriptive. No need for a "full example" if the sub-options are properly documented.

Reply:
Tried to address it at bd708da. Let me know if I got the suggestion correctly :)

Reply:
I was thinking more along the lines of "now we have too many options for `redact`, so its description needs to be split" => that would apply to the whole thing, not just the new sub-keys added in this PR.

So basically the `redact` documentation would be stripped down to just `# -- Configures the redact options for Hubble.` and then the rest of the docs moved to the appropriate subkeys, akin to what's done for the `metrics` entry right above `redact`.

Does this make sense?

Reply:
Allright got it! Pushed [41675df](https://github.com/cilium/cilium/pull/26724/commits/41675df9f8244094603a3a2673ad79e5b3d8d7a3) to apply this :)

Reply:
Thanks, made some comments below.